presto-cli/src/main/java/com/facebook/presto/cli/WarningsPrinter.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.cli;
+
+import com.facebook.presto.spi.PrestoWarning;
+
+import java.util.List;
+
+public interface WarningsPrinter
+{
+    void print(List<PrestoWarning> warnings);
+
+    void printWithSeparators(List<PrestoWarning> warnings);
+
+    // Only print separator for initial warnings
+    void printWithInitialSeparator(List<PrestoWarning> warnings);
```

Comment:
This method prints an initial separator and warnings, but the javadoc suggests this only prints a separator.